Makefile

```diff
@@ -55,11 +55,14 @@ cluster-prepare:
 	$(MAKE) -C third_party/cert-manager deploy
 	$(MAKE) -C third_party/registry deploy
 	$(MAKE) -C third_party/vault deploy
+	kubectl apply -f https://raw.githubusercontent.com/IBM/dataset-lifecycle-framework/master/release-tools/manifests/dlf.yaml
```

Comment:
Yes, it will work on openshift, although they have a different installation for the openshift, more security oriented, I think. I'll open a discussion in the channel.